friends/src/Components/FriendList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+const FriendList = (props) => {
+	return (
+		<div>
+			{props.friends.map((friend) => (
+				<ul key={friend.id}>
+					<button type="submit" onClick={(props) => props.updateFriend(friend.id)}>
+						{friend.updateFriend}
```

Comment:
Not related to your bug, but there's no `updateFriend` or `deleteFriend` values on the `friend` data object so you can take out this line and line 15.